api-flask/app/sample_requests.py

```diff
@@ -0,0 +1,116 @@
+        "legend": [
+            {
+                "label": "-0",
+                "color": "#ffffff",
+                "alpha": 0
+            },
+            {
+                "label": "1%",
+                "color": "#d79b0b"
+            },
+            {
+                "label": "60%",
+                "color": "#e1b344"
+            },
+            {
+                "label": "80%",
+                "color": "#ebcb7d"
+            },
+            {
+                "label": "90%",
+                "color": "#f5e3b6"
+            },
+            {
+                "label": "110%",
+                "color": "#f2f2f2"
+            },
+            {
+                "label": "120%",
+                "color": "#b5e7fe"
+            },
+            {
+                "label": "140%",
+                "color": "#7dd4fd"
+            },
+            {
+                "label": "180%",
+                "color": "#45c1fc"
+            },
+            {
+                "label": "> 180%",
+                "color": "#0fb0fb"
+            }
+        ]
+    },
+    "zones": {
+        "type": "FeatureCollection",
+        "name": "admin_boundaries",
+        "crs": {
+            "type": "name",
+            "properties": {
+                "name": "urn:ogc:def:crs:OGC:1.3:CRS84"
+            }
+        },
+        "features": [
+            {
+                "type": "Feature",
+                "properties": {
+                    "OBJECTID": 277,
+                    "ST": "Ayeyarwady",
+                    "ST_PCODE": "MMR017",
+                    "DT": "Pyapon",
+                    "DT_PCODE": "MMR017D006",
+                    "TS": "Bogale",
+                    "TS_PCODE": "MMR017024",
+                    "SELF_ADMIN": None,
+                    "ST_RG": "Region",
+                    "TS_MMR4": "ဘိုကလေး",
+                    "AREA": 1,
+                    "DT_MMR4": "ဖျာပုံခရိုင်",
+                    "mmr_polbnd": "ဧရာဝတီတိုင်းဒေသကြီး"
+                },
+                "geometry": {
+                    "type": "MultiPolygon",
+                    "coordinates": [
+                        [
+                            [
+                                [
+                                    95.3170445470002,
+                                    15.928591091000044
+                                ],
+                                [
+                                    95.31199007400005,
+                                    15.926271754000084
+                                ],
+                                [
+                                    95.3115704550001,
+                                    15.928167658000064
+                                ],
+                            ]
+                        ]      
+                    ]
+                }
+            }
+        ]
+    },
+    "email": "vik@gmail.com",
+    "prism_url": "https://prism-mongolia.org"
+}
```

Comment:
"/n"

Reply:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline